src/components/Sidebar.js

```diff
@@ -1,26 +1,223 @@
-import React from 'react'
+import React from "react";
+import { useSelector } from "react-redux";
 
 const Sidebar = () => {
+  const isMenuOpen = useSelector((store) => store.toggle.isMenuOpen);
+  
+  const sidebarItems = [
```

Comment:
move this to a new file in constants folder

Reply:
isMenuOpen should be moved?